src/authorize/check.rs

```diff
@@ -45,7 +45,14 @@ async fn token_info_request(ac_token: String) -> Result<Response, reqwest::Error
 
 // check if current access token is valide.
 // if not generate new access token
-pub async fn check_token_validity(mut ac_token: String) -> Result<(String, token::TokenInfo)> {
+pub async fn check_token_validity(session: Session) -> Result<(String, token::TokenInfo)> {
+    // let mut response = token_info_request(ac_token.to_owned()).await?;
+
+    // let mut ac_token = session.access_token.to_owned();
+    let mut ac_token = match &session.access_token {
+        Some(x) => x.to_owned(),
+        None => String::new(),
+    };
```

Comment:
```suggestion
    let ac_token = session.access_token.map(|x| x.to_owned()).unwrap_or_default();
```
이런 식으로 쓸 수도 있습니다. 그런데 기본값으로 String::new를 넣기보다는 토큰의 타입이 Option<String>인 편이 더 나을거에요.

Reply:
> 이런 식으로 쓸 수도 있습니다. 그런데 기본값으로 String::new를 넣기보다는 토큰의 타입이 Option인 편이 더 나을거에요.
네! 감사합니다.